api.planx.uk/send/uniform.js

```diff
@@ -419,14 +416,19 @@ async function retrieveSubmission(token, submissionId) {
 /**
  * Helper method to locally download S3 files, add them to the zip, then clean them up
  *
- * @param {string} filePath - s3 `path/key` to file
+ * @param {string} url - our file URL, eg api.planx.uk/file/private/path/key
  * @param {string} path - file name for download
  * @param {string} folder - AdmZip archive
  */
-const downloadFile = async (filePath, path, folder) => {
-  const { body } = await getFileFromS3(filePath);
+const downloadFile = async (url, path, folder) => {
+  const res = await fetch(url, { headers: { 'api-key': process.env.FILE_API_KEY } });
```

Comment:
interesting to note that without the header, the files were actually still showing up in the zip, but could not be opened. that may make for unpleasant troubleshooting in the future?

Reply:
good shout. we probably want to check `res.ok`

https://developer.mozilla.org/pt-BR/docs/Web/API/Fetch_API/Using_Fetch#Checking_that_the_fetch_was_successful

Reply:
addressed this in latest push !

submissions on the pizza are failing with the below error (but succeeding locally) - so we're going to ensure those are sorted before any merge but thanks for initial phone review :upside_down_face: 
```
"message": "FetchError: request to https://api.1278.planx.pizza/file/private/yjki3e20/Screenshot%20from%202022-11-15%2015-18-51%281%29.png failed, reason: connect ECONNREFUSED 127.0.1.1:443\"}
```